spec/Section 5 -- Validation.md

```diff
@@ -1424,6 +1466,103 @@ arguments, an input object may have required fields. An input field is required
 if it has a non-null type and does not have a default value. Otherwise, the
 input object field is optional.
 
+### Oneof Input Objects Have Exactly One Field
+
+**Formal Specification**
+
+* For each {operation} in {document}:
+  * Let {oneofInputObjects} be all Oneof Input Objects transitively included in the {operation}.
+  * For each {oneofInputObject} in {oneofInputObjects}:
+    * Let {fields} be the fields provided by {oneofInputObject}.
+    * {fields} must contain exactly one entry.
+    * Let {field} be the sole entry in {fields}.
+    * Let {value} be the value of {field}.
+    * {value} must not be the {null} literal.
+    * If {value} is a variable:
+      * Let {variableName} be the name of {variable}.
+      * Let {variableDefinition} be the {VariableDefinition} named {variableName} defined within {operation}.
+      * Let {variableType} be the expected type of {variableDefinition}.
+      * {variableType} must be a non-null type.
+
+**Explanatory Text**
+
+Oneof Input Objects require that exactly one field must be supplied and that
+field must not be {null}.
+
+An empty Oneof Input Object is invalid.
+
+```graphgl counter-example
```

Comment:
you wrote `graphgl` instead of `graphql` for all of these example snippets

Reply:
Hawk eyes! :raised_hands: